binary-workflows/Readme.md

```diff
@@ -8,17 +8,17 @@
 
 - [Summary](#summary)
 - [Query Engine](#query-engine)
-  - [Development Installation](#development-installation)
-    - [OSX, Windows](#osx-windows)
-  - [Production Installation](#production-installation)
-    - [Lambda, Serverless, Apex, Netlify Functions, rsync](#lambda-serverless-apex-netlify-functions-rsync)
+  * [Development Installation](#development-installation)
```

Comment:
Curious: What tool were you using that caused the "fix"/replacement of `-` to `*` etc?

Reply:
Ah good point, I think it was my prettier for Markdown in VS Code that applied this formatting. We also use [`markdown-toc`](https://www.npmjs.com/package/markdown-toc) to generate TOCs (see [here](https://github.com/prisma/specs/blob/master/Makefile#L14))

https://github.com/prisma/specs/blob/master/Makefile#L14

Reply:
Yeha, which is why I was asking: This looked like it would be undone by build logic probably.